.changeset/odd-cats-act.md

```diff
@@ -0,0 +1,14 @@
+---
+"@salt-ds/icons": minor
+---
+
+Added `saltIcons.css` with all icons SVG as background image.
+
+```js
+import "@salt-ds/icons/saltIcons.css";
```

Comment:
checking since it's copied to salt-icons.css, is saltIcons.css the one consumers would use or would it be salt-icons?

Reply:
i removed the copy for it to work how country symbols do for now